### DISCUSSION TITLE
Windows nightly is broken

### DISCUSSION THREAD

**Discussion by mvorisek** [apache/netbeans#6788] [General]
https://ci-builds.apache.org/job/Netbeans/job/netbeans-windows/

Please fix it, it is broken for more than 7 months and no prebuild artifacts are available therefore.

related with https://github.com/apache/netbeans/pull/6712#issuecomment-1838208676

**Comment by neilcsmith-net** [comment]
Please don't screw around with the issue template.

You're looking in the wrong place for a nightly build of NetBeans.  See https://netbeans.apache.org/front/main/download/dev/ and use the download from the netbeans-linux job.

**Reply by mvorisek** [reply]
So the Windows build is deprecated? If so, please document it on https://ci-builds.apache.org/job/Netbeans/job/netbeans-linux/ as there is:

> There is also a [windows version](https://ci-builds.apache.org/job/Netbeans/job/netbeans-windows) of this job.

I am especially interested in the Windows version because of Windows installer.

**Reply by neilcsmith-net** [reply]
No, the build _on_ Windows there is not currently working.  Building on Windows is not deprecated, and is tested with every release.  We are discussing rationalisation of CI, so that job might be removed.  It is not relevant to anyone wanting to use a nightly build. 

There is no nighty installer.  You need to use the zip bundle to use a nightly.

To add, on the issue form - any junk you fill in to bypass the form is emailed to every NetBeans developer!

**Reply by mvorisek** [reply]
Thank you for the explanation and sorry for the stir.
